### PR TITLE
Add New Featured Artist: Ritorikal news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2022-09-24-new-featured-artist-ritorikal.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/305/Imagine/Ritorikal%20-%20Imagine.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+Mappers need music to create beatmaps. [**Ritorikal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/305) has music. You're a mapper. You can put two and two together.
+
+One more artist is coming this week, and they're a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Saturday for all the juice.
```
Comment (Nivalyx) [inline news/2022-09-24-new-featured-artist-ritorikal.md:55]:
```suggestion
One more artist is coming this week, and they're a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Wednesday for all the juice.
```

Comment (Nivalyx) [reply news/2022-09-24-new-featured-artist-ritorikal.md:55]: ...actually shouldn't this be something like

```suggestion
Our next artist is a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Wednesday for all the juice.
```

File: news/2022-09-24-new-featured-artist-ritorikal.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+date: 2022-09-24 10:00:00 +0000
+---
+
+We're proud to welcome **Ritorikal** to our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/305/header.jpg)
+
+Merging various influences, genres, and his own unending curiosity, producer **Ritorikal** has been generating tunes since as far back as 2009. While electronic music is his shtick, he tends to pull in more organic sounds on top (primarily strings, keys, and guitars), making for a less conventional EDM vibe.
+
+Oh, and **Ritorikal** is also [an osu! player](https://osu.ppy.sh/users/8061786), which miiiight relate to a few of his influences like *MitiS*, *Seven Lions*, [*Koven*](https://osu.ppy.sh/beatmaps/artists/293), and *Virtual Riot*.
```
Comment (peppy) [inline news/2022-09-24-new-featured-artist-ritorikal.md:13]: The "which might relate to" doesn't make much sense to me. I don't know what it's trying to say.

Comment (Ephemeralis) [reply news/2022-09-24-new-featured-artist-ritorikal.md]: I've slapped a reword on this to better articulate the (probable) intent of it, at least as far as I understand it.

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**